docs/source/deployment/deploy_sre.md

```diff
@@ -35,31 +53,39 @@ sre:
   software_packages: # any/pre-approved/none: which packages from external repositories to allow
   timezone: # Timezone in pytz format (eg. Europe/London)
   workspace_skus: # List of Azure VM SKUs - see cloudprice.net for list of valid SKUs
-```
+:::
+
+::::
 
 ## Upload the configuration file
 
 - Upload the config to Azure. This will validate your file and report any problems.
 
-```{code} shell
-$ dsh config upload config.yaml
-```
+:::{code} shell
+$ dsh config upload PATH_TO_YOUR_EDITED_YAML_FILE
+:::
+
+:::{hint}
+If you want to make changes to the config, edit this file and then run `dsh config upload` again
+:::
 
 ## Requirements
 
 :::{important}
 As private endpoints for flexible PostgreSQL are still in preview, the following command is currently needed:
 
-```{code} shell
-$ az feature register --name "enablePrivateEndpoint" --namespace "Microsoft.DBforPostgreSQL"
-```
+:::{code} shell
+$ az feature register --name "enablePrivateEndpoint" \
+                      --namespace "Microsoft.DBforPostgreSQL" \
+                      --subscription NAME_OR_ID_OF_YOUR_SUBSCRIPTION
+:::
```

Comment:
note - I just tried this and got the following error:
```
az feature register --name "enablePrivateEndpoint" --namespace "Microsoft.DBforPostgreSQL" --subscription "Data Safe Haven Development"
Once the feature 'enablePrivateEndpoint' is registered, invoking 'az provider register -n Microsoft.DBforPostgreSQL' is required to get the change propagated
(FeatureNotFound) The feature 'enablePrivateEndpoint' could not be found.
Code: FeatureNotFound
Message: The feature 'enablePrivateEndpoint' could not be found.
```

Reply:
Yeah - David SJ found the same thing. I think this might be deprecated/not needed now. Worth removing from the docs?